gulpfile.js

```diff
@@ -18,7 +18,7 @@ gulp.task('compress-css', function () {
 
 // JS COMPRESS
 async function compress_all_js() {
-    await gulp.src(['js/src/*.js', '!js/src/apbct-public--*.js'])
+    await gulp.src(['js/src/*.js', '!js/src/apbct-public--*.js', 'js/src/apbct-public--3--cleantalk-modal.js'])
```

Comment:
This looks like some a some mistake, are you sure that this is productive file name you want?

Reply:
Yes I'm sure. And I'm sure this is not beautiful filename. But for now I haven't other way.